src/init/features/genkit/index.ts

```diff
@@ -66,7 +66,7 @@ async function getGenkitVersion(): Promise<GenkitInfo> {
     genkitVersion = process.env.GENKIT_DEV_VERSION;
   } else {
     try {
-      genkitVersion = await spawnWithOutput("npm", ["view", "genkit", "version"]);
+      genkitVersion = await spawnWithOutput("npm", ["view", "genkit-cli", "version"]);
```

Comment:
that's not right either... downstream you use this version to install both `genkit` and `genkit-cli` packages... 
We need separate `getGenkitCLIVersion()` and `getGenkitRuntimeVersion()` functions.

Reply:
We only install 'genkit'  if the version is < 0.6.0. That can only happen when the environment variable is used to override the latest. So looking for genkit-cli if the environment variable isn't set is the correct behaviour since it will always be > 0.6.

Reply:
Also when do we want to drop support for the super old 0.6.0 and below?